### DISCUSSION TITLE
Add `isDir` template function

### DISCUSSION THREAD

**Discussion by simonsystem** [helmfile/helmfile#1656] [Ideas]
When I look into the documentation, I can find the `isFile` function here: https://helmfile.readthedocs.io/en/latest/templating_funcs/#isfile

But what I'm missing is a proper way to check if a directory exists, the same way.

Currently, for checking if `./subdirectory` exists, I have to do:

```
{{- $subdirectoryExists := false }}
{{- range $index,$item := readDirEntries "." }}
{{-   if eq $item.Name "subdirectory" | and $item.IsDir }}
{{-     $subdirectoryExists = true }}
{{-   end }}
{{- end }}

{{- if $subdirectoryExists }}
do whatever you want to
{{- end }}
```

Let's create a proper `isDir` function:

```
{{- if isDir "./subdirectory" }}
do whatever you want to
{{- end }}
```

**Comment by yxxhero** [comment]
@simonsystem PR is welcome.

**Comment by simonsystem** [comment]
Created a PR: https://github.com/helmfile/helmfile/pull/1657
